Week2/ramazanInce.js

```diff
@@ -0,0 +1,158 @@
+console.log(`
+Question 1`);
+// 1. Write a console.log statement saying
+// "Hello World!" for each language that you know.
+
+console.log(`Hello World
+Merhaba Dunya
+Priviyet Mir
+Marhaban Dunya
+Hallo Wereld
+`);
+
+console.log(`
+Question 2`);
+// 2. Consider the following code:
+
+console.log("I'm awesome");
+
+console.log(`
+Question 3`);
+// 3. Declare a variable x and initialize it with
+// an integer, using these exact steps:
+// 3.1 First, declare your variable x (do not initialize it yet).
+// 3.2 Add a console.log statement that explains in
+// words what you think the value of x is, like in this example:
+
+var x;
+console.log('the value of my variable x will be : whateverYouThinkItWillLog');
+console.log(x);
+x = 41;
+console.log('the value of x is my age');
+console.log(x);
+
+console.log(`
+Question 4`);
+// 4. Declare a variable y and assign a string to it.
+var y = `Leersum`;
+console.log(`the value of y is the city where I stay`);
+console.log(y);
+y = `Netherlands`;
+console.log(`the value of y is the country where I stay`);
+console.log(y);
+
+console.log(`
+Question 5`);
+//5. How do you round the number 7.25, to
+//the nearest integer (i.e., whole number)?
+
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+let b = Math.max(a, z);
```

Comment:
I really liked your usage of `Math.max` to find out the max of `a` and `z` 🙂 

But you could have used `const` keyword here instead of `let` as we are never reassigning `a`, `b` or `z`, isn't it?

Reply:
Through out the homework I try to use different letter not to have a problem since it was in one page. But for the followings works I will do in that way. Thanks